components/brave_ads/common/features.cc

```diff
@@ -16,6 +16,9 @@ const base::Feature kAdNotifications{"AdNotifications",
 const base::Feature kCustomAdNotifications{"CustomAdNotifications",
                                            base::FEATURE_DISABLED_BY_DEFAULT};
 
+const base::Feature kRequestAdsEnabledApi{"RequestAdsEnabledApi",
+                                          base::FEATURE_DISABLED_BY_DEFAULT};
```

Comment:
disabled by default?

Reply:
Changed to enable by default. 
Think need to ask from @jsecretan how we will turn on this feature. 
Should we turn it on on 100% from code or should we  turn it on gradually using griffin?

Reply:
I don't think we need to Griffin flag this one because that would probably get confusing in any case.